scripts/script_requirements.txt

```diff
@@ -0,0 +1,2 @@
+click
+PyGithub
```

Comment:
OK - I'm fine with using `github3`. I didn't know there was an existing standard - I'll convert over to it.